service/src/test/java/io/nuvalence/workmanager/service/service/NoteServiceTest.java

```diff
@@ -128,7 +130,7 @@ public void testCreateTransactionNote() {
         String testName = "testName";
         NoteType noteType = new NoteType(UUID.randomUUID(), testName);
 
-        when(noteTypeRepository.findByName(testName)).thenReturn(Optional.of(noteType));
+        when(noteTypeRepository.findByName(eq(testName))).thenReturn(Optional.of(noteType));
```

Comment:
this eq here is linted by SonarQube to be not needed. Not a big deal but removing it helps avoid adding a new SQ lint
```suggestion
        when(noteTypeRepository.findByName(testName)).thenReturn(Optional.of(noteType));
```

Reply:
thanks Dan, I'll try to keep these things in mind for future PRs, this is done